org.eclipse.scout.rt.rest.test/src/test/java/org/eclipse/scout/rt/rest/client/proxy/ErrorDoRestClientExceptionTransformerTest.java

```diff
@@ -56,6 +56,24 @@ public void testTransformStatusSuccessful() {
     assertEquals(mockCause, m_exceptionTransformer.transform(mockCause, mockResponse(Status.OK)));
   }
 
+  @Test
+  public void testWithoutErrorDo() {
+    String message = "mock";
+    RuntimeException mockCause = new RuntimeException(message);
+    ErrorResponse errorResponse = BEANS.get(ErrorResponse.class);
+    RuntimeException actualException = m_exceptionTransformer.transform(mockCause, mockResponse(Status.FORBIDDEN, errorResponse));
+    assertProcessingException(AccessForbiddenException.class, mockCause, null, 0, actualException);
+    assertEquals(message + " [severity=ERROR]", actualException.getMessage());
+  }
+
+  @Test
+  public void testWithoutException() {
+    ErrorResponse errorResponse = BEANS.get(ErrorResponse.class);
+    RuntimeException actualException = m_exceptionTransformer.transform(null, mockResponse(Status.FORBIDDEN, errorResponse));
```

Comment:
Add another testcase for defaultTransform(), e.g. using a server error like Status.INTERNAL_ERROR